scripts/check.sh

```diff
@@ -1,7 +1,41 @@
 #!/bin/bash
-
+set -e
```

Comment:
For consistency, should we add this to all scripts to quit on errors?
(Also, line 3 was removed or is absent in other files, so could be removed here, too)